### PR TITLE
BugFix: Cache Netelements correctly

### PR DESCRIPTION
**Pull request by cschra**

### CODE REVIEW

File: modules/HfcReq/Entities/NetElement.php [hunk 1]
```diff
@@ -519,7 +518,7 @@ public function updating($netelement)
             return;
         }
 
-        if ($netelement['original']['parent_id'] != $netelement['attributes']['parent_id']) {
+        if ($netelement->isDirty('parent_id') || $netelement->isDirty('name')) {
```
Comment (NinoRy) [inline modules/HfcReq/Entities/NetElement.php]: this loops twice through all columns ... store in variable first or just leave it as is for that single case

File: modules/HfcReq/Entities/NetElement.php [hunk 2]
```diff
@@ -518,7 +518,7 @@ public function updating($netelement)
             return;
         }
 
-        if ($netelement->isDirty('parent_id') || $netelement->isDirty('name')) {
+        if ($netelement->isDirty('parent_id', 'name')) {
```
Comment (NinoRy) [inline modules/HfcReq/Entities/NetElement.php]: multiple use of isDirty again - pls store in variable

File: modules/HfcReq/Entities/NetElement.php [hunk 3]
```diff
@@ -529,14 +529,14 @@ public function updating($netelement)
...
+        if ($netelement->isDirty('netelementtype_id')) {
             $new_params = $netelement->netelementtype->parameters;
 
-            foreach ($netelement->indices as $indices) {
-                // assign each indices of parameter to new parameter with same oid
-                if ($new_params->contains('oid_id', $indices->parameter->oid->id)) {
-                    $indices->parameter_id = $new_params->where('oid_id', $indices->parameter->oid->id)->first()->id;
-                    $indices->save();
+            foreach ($netelement->indices as $indice) {
+                // assign each indice of parameter to new parameter with same oid
```
Comment (NinoRy) [inline modules/HfcReq/Entities/NetElement.php]: why do you change this ? each entry contains really multiple indices

File: modules/HfcReq/Entities/NetElement.php (hunk 1 above)
Comment (cschra) [reply modules/HfcReq/Entities/NetElement.php]: as discussed, this is fixed later on

**Pull request merged**